src/ops/fuelup_toolchain/uninstall.rs

```diff
@@ -36,8 +36,9 @@ pub fn uninstall(command: UninstallCommand) -> Result<()> {
                 }
 
                 error!(
-                "Could not set default toolchain after uninstallation of currently used toolchain. 
-                Please run `fuelup default <toolchain>` to manually switch your current toolchain."
+                    "{}\r\t{}",
+                    "Could not set default toolchain after uninstallation of currently used toolchain",
```

Comment:
It seems like we shouldn't let the user uninstall their current default toolchain. What do you think?

Reply:
Yeah I agree with Sophie. There should always be at least one toolchain installed and that should be the default.

Reply:
Instead of making sure there is only one toolchain left, it should always be the default toolchain.